src/Angor/Client/Pages/Wallet.razor

```diff
@@ -594,7 +594,16 @@
 
             var wallet = _walletStorage.GetWallet();
 
-            return await _walletOperations.SendAmountToAddress(wallet, _sendInfo);
+            var res = await _walletOperations.SendAmountToAddress(wallet, _sendInfo);
+
+            if (res.Success)
+            {
+                _walletOperations.UpdateAccountInfoWithSpentTransaction(localAccountInfo, res.Data, _sendInfo.ChangeAddress);
+                localAccountInfo.CalculateBalance();
```

Comment:
This does not return anything

Reply:
It sets the TotalBalance and TotalUnConfirmedBalance properties that are used in the page